### ISSUE TITLE
Training PP-Lite-Seg using custom data

### ISSUE THREAD

**Issue by MaxEAB** [Deci-AI/super-gradients#1039]
### 💡 Your Question

Hi, How do I set up my custom data for retraining or fine-tuning PP-Lite-Seg? Is there any documentation or an example dataset that describes how to do this? 

As per [this](https://github.com/Deci-AI/super-gradients/blob/master/tutorials/SuperGradients_Semantic_Segmentation_How_to_Connect_Custom_Dataset.ipynb), it looks like the folder needs a text file containing file names. But it is not clear.

Thanks,
